dashboard/app/models/levels/level.rb

```diff
@@ -803,6 +803,13 @@ def summarize_for_lesson_show(can_view_teacher_markdown)
     }
   end
 
+  def summarize_for_lab2_properties
```

Comment:
nit: mind adding a doc for reference here?

Reply:
Good call; done.

Reply:
Oh, and just realized, we'd need to update `levels_controller` with this as well right?

Reply:
Good point.  Just fixed in parent branch.

As it happens, the individual level pages still worked fine because we didn't change what `level_properties` returns for a music level, and we're using the legacy server-rendered page for a standalone video.